pkl-gradle/src/main/java/org/pkl/gradle/TaskConstants.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright © 2024 Apple Inc. and the Pkl project authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.pkl.gradle;
+
+/** Constant values used by tasks */
+class TaskConstants {
```

Comment:
I believe it is more conventional to put these constants directly to the plugin class.

Reply:
What about the category settings, which must be shared between plugins? I think there may be symbols for those somewhere in Gradle's API. I'll look for appropriate ones to use.

Reply:
There's [LifecycleBasePlugin](https://github.com/gradle/gradle/blob/master/platforms/software/platform-base/src/main/java/org/gradle/language/base/plugins/LifecycleBasePlugin.java) which provides the base task names and groups. 

Then there's [JvmConstants](https://github.com/gradle/gradle/blob/master/platforms/jvm/platform-jvm/src/main/java/org/gradle/api/internal/tasks/JvmConstants.java), but this seems less appropriate for us.

I think it's fine to just have these as strings, though.

Reply:
There is only one plugin in the project, so there is nothing to share, or an I missing something?

Anyway, if you look at the built-in Gradle plugins, they always contain constants related to tasks configured by these plugins within the plugin class. If we ever decide to make new plugins, we can move these constants to the appropriate location at that time.